lib/interface/login/login.dart

```diff
@@ -124,40 +144,29 @@ class LoginScreenState extends State<LoginScreen>
                 child: new TextFormField(
                   onSaved: (val) => _url = val,
                   decoration: new InputDecoration(
-                      labelText: MyLocalizations.of(context, 'URL')),
+                    labelText: MyLocalizations.of(context, 'URL'),
+                    border: OutlineInputBorder(),
+                  ),
                 ),
               ),
-              _normalDown()
+              _normalDown(),
+              SizedBox(
+                height: 15,
+              )
             ],
           ),
         ),
         _isLoading ? new CircularProgressIndicator() : loginBtn
       ],
-      crossAxisAlignment: CrossAxisAlignment.center,
     );
 
     return new Scaffold(
       appBar: null,
       key: scaffoldKey,
-      body: new Container(
```

Comment:
Using a Container as a top-level widget is a good practice as it can be decorated with a BoxDecoration, such as a background, a border, or a shadow. Also, it can have margins, padding, and constraints applied to its size.